target/linux/ramips/mt7620/config-4.14

```diff
@@ -200,6 +200,8 @@ CONFIG_RALINK_WDT=y
 # CONFIG_RCU_STALL_COMMON is not set
 CONFIG_REGMAP=y
 CONFIG_REGMAP_MMIO=y
+CONFIG_REGULATOR=y
```

Comment:
This should be broken out as a separate PR

Reply:
OK